src/data/api.js

```diff
@@ -19,13 +17,17 @@ export const getValidationApiUrl = (service) => `${getApiBaseUrl()}/plugin-cvw/a
  */
 
 export async function getCoursesByUser() {
-  const { data } = await getAuthenticatedHttpClient()
-    .get(getCoursesApiUrl(), {
+  try {
```

Comment:
I am not sure the try/catch implementation makes sense here

Reply:
The other way that I think is to apply them in each slice to keep the benefits that the ```builder.addCase``` method give us with the loading statuses

![image](https://github.com/eduNEXT/frontend-app-validation-panel/assets/86393372/5f223fc5-77c6-44ca-8621-90a3c72feba4)

Reply:
Working in the connection to the backend I realized that is not necessary at all the try/catch block for making use of the benefit I was talking about before; it looks like the getAuthenticatedHttpClient method sends the error or the response. I'll send the change without the try/catch blocks in a few seconds.

Reply:
Lets apply that changes by now